Solutions/CiscoSecureEndpoint/Analytic Rules/CiscoEndpointHighAlert.yaml

```diff
@@ -0,0 +1,38 @@
+id: 4683ebce-07ad-4089-89e3-39d8fe83c011
+name: Cisco SE High Events Last Hour
+description: |
+  'Find events from Cisco Secure Endpoint that are of High severity in the last hour.'
+severity: High
+requiredDataConnectors:
+  - connectorId: CiscoSecureEndpoint
+    dataTypes:
+      - CiscoSecureEndpoint_CL
+queryFrequency: 1h
+queryPeriod: 1h
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - Execution
+  - InitialAccess
```

Comment:
Do we want to add the relevantTechniques: section after this.

Reply:
We can if you want, but it's going to be a long list - since High alerts can be everything from executed malware to CnC server access, it could be hard to identify specific ones.